core/arch/arm/include/kernel/tz_ssvce.h

```diff
@@ -57,13 +57,11 @@ static inline void tlbi_mva_allasid_nosync(vaddr_t va)
 #ifdef ARM64
 	uint64_t mva = va;
```

Comment:
From data size, it should not, but won't strict aliasing check fail on the types ?
If current build without cast runs ok, i will update the change according to your suggestion..